api/proto/teleport/presence/v1/relay_server.proto

```diff
@@ -0,0 +1,40 @@
+// Copyright 2025 Gravitational, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package teleport.presence.v1;
+
+import "teleport/header/v1/metadata.proto";
+
+option go_package = "github.com/gravitational/teleport/api/gen/proto/go/teleport/presence/v1;presencev1";
+
+// A heartbeat for a relay service; this message serves as both the type used in
+// the v1 service and as the canonical v1 storage format (in protojson).
+message RelayServer {
+  // fixed string, "relay_server".
+  string kind = 1;
+  // fixed string, "".
+  string sub_kind = 2;
+  // fixed string, "v1".
+  string version = 3;
+
+  teleport.header.v1.Metadata metadata = 4;
+
+  // resource spec
+  message Spec {
```

Comment:
I guess the reason why you added the spec inside the message was because users aren't supposed to create/manage these, right?